runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -247,6 +247,18 @@ public void onChatMessage(ChatMessage event)
 				return;
 			}
 		}
+		
+		if (config.screenshotPet())
+		{
+			switch (chatMessage)
+			{
+				case "You have a funny feeling like you're being followed":
+				case "You feel something weird sneaking into your backpack":
+				case "You have a funny feeling like you would have been followed":
+					String fileName = "Pet " + TIME_FORMAT.format(new Date());
```

Comment:
Why use different format than kingdom screenshot for filename?

Reply:
Receiving more than one pet one day would override the previous images

Reply:
Btw, don't merge this. I need to make it conatins instead of equeals

Reply:
Ok, it should be fine now